### DISCUSSION TITLE
weaver gives such an error.

### DISCUSSION THREAD

**Discussion by Akrepkapo** [IBAX-io/go-ibax#2048] [Bug Submission]
weaver gives such an error. @ibaxio @AssKyle @group-monkey 
![Screenshot_2022-09-29-23-44-28-24_40deb401b9ffe8e1df2f1cc5ba480b12](https://user-images.githubusercontent.com/96292241/193138870-5a2b0380-675e-4faa-afe4-b401d767e44a.jpg)

**Comment by yolandadadada** [comment]
Network deployment in progress

**Comment by Akrepkapo** [comment]
Okey
